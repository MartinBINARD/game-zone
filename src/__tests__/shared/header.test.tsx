import ShallowRenderer from 'react-shallow-renderer';
import Header from 'src/shared/header';

describe('header component', () => {
  it('should be render', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<Header title="GameZone" />);
    expect(wrapper).toMatchSnapshot();
  });
});
