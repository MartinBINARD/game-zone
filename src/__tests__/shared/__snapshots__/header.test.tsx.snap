// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component should be render 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "row",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      {
        "height": 26,
        "marginHorizontal": 10,
        "width": 26,
      }
    }
  />
  <Text
    style={
      {
        "color": "#333",
        "fontSize": 20,
        "fontWeight": "bold",
        "letterSpacing": 1,
      }
    }
  >
    GameZone
  </Text>
</View>
`;
